src/main/scala/com/databricks/sparkdl/Models.scala

```diff
@@ -45,153 +43,153 @@ object Models {
         }
     }
   }
-  
+
   /**
-   * Model provided by Keras. All cotributions by Keras are provided subject to the
-   * MIT license located at https://github.com/fchollet/keras/blob/master/LICENSE
-   * and subject to the below additional copyrights and licenses.
-   *
-   * Copyright 2016 The TensorFlow Authors.  All rights reserved.
-   *
-   * Licensed under the Apache License, Version 2.0 (the "License");
-   * you may not use this file except in compliance with the License.
-   * You may obtain a copy of the License at
-   *
-   * http://www.apache.org/licenses/LICENSE-2.0
-   *
-   * Unless required by applicable law or agreed to in writing, software
-   * distributed under the License is distributed on an "AS IS" BASIS,
-   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   * See the License for the specific language governing permissions and
-   * limitations under the License.
-   */
+    * Model provided by Keras. All cotributions by Keras are provided subject to the
```

Comment:
Nit: we use Javadoc style in our comments instead of Scaladoc, so the original indentation was correct - see the [DB style guide](https://github.com/databricks/scala-style-guide#documentation-style). AFAICT regenerating this file with `gen_app_models.py` should give you a file with the right indentation?